pages/service/index.vue

```diff
@@ -16,7 +15,8 @@ const config = useRuntimeConfig();
 const { data: response } = await useFetch(
   `${config.public.kurocoApiDomain}/rcms-api/1/content/details/service`,
   {
-    credentials: 'include',
+    credentials: "include",
+    server: false,
   }
-);
+).catch((error) => console.info(error));
```

Comment:
reason?

Reply:
No reason. Not necessary to have so I'll remove it.